src/aragonjs-wrapper.js

```diff
@@ -227,12 +240,35 @@ const subscribe = (
     connectedApp: null,
     connectedWorkers: workerSubscriptionPool,
 
-    apps: apps.subscribe(apps => {
+    apps: apps.subscribe(async apps => {
+      const storageApp = apps.find(({ name }) => name === 'Storage')
+      let homeAppName = ''
+      let homeAppAddr = ''
+      if (storageApp && storageApp.proxyAddress) {
+        const account = await getMainAccount(wrapper.web3)
+
+        homeAppAddr = await AragonStorage.get(
+          wrapper.web3,
+          storageApp.proxyAddress,
+          account,
+          'HOME_APP'
+        )
+
+        homeAppName = await AragonStorage.get(
```

Comment:
we can save 1 transaction on set, if we concatenate both the address and the name into one `_value` string, since the address will always be fixed width is easy to do

Reply:
then performing the split here...

Reply:
Ur right... i didn't even think about that.